bucket/clash.net.json

```diff
@@ -0,0 +1,26 @@
+{
+    "homepage": "https://github.com/ClashDotNetFramework/ClashDotNetFramework",
+    "description": "A Clash GUI Proxy For Windows Based On .NET 5",
```

Comment:
Keeping `For Windows` and dotnet version is not needed. 

Scoop is only for windows and version of dotnet is not relevant